tests/python/PyImportSortBearTest.py

```diff
@@ -20,4 +20,5 @@
     ('import xyz\nimport abc\n',
      'import abc\nimport xyz\n'),
     settings={'known_standard_library_imports': 'xyz',
-              'known_first_party_imports': 'abc'})
+              'known_first_party_imports': 'abc',
+              'known_third_party_imports': 'datetime'})
```

Comment:
You are asking isort to group `datetime` with the third party imports. Therefore add `datetime` with the third party imports in the good file and in another group in the bad file.